pilot/pkg/features/pilot.go

```diff
@@ -701,6 +701,12 @@ var (
 	UseCacertsForSelfSignedCA = env.Register("USE_CACERTS_FOR_SELF_SIGNED_CA", false,
 		"If enabled, istiod will use a secret named cacerts to store its self-signed istio-"+
 			"generated root certificate.").Get()
+
+	GatewayAPIDefaultGatewayClass = env.Register("PILOT_GATEWAY_API_DEFAULT_GATEWAYCLASS", "istio",
+		"Name of the default GatewayClass").Get()
+
+	ManagedGatewayController = env.Register("PILOT_GATEWAY_API_CONTROLLER_NAME", "istio.io/gateway-controller",
+		"Gateway API controller name. istiod will only reconcile Gateway API resources referencing a GatewayClass with this controller name").Get()
```

Comment:
These two envs are too confusing to expose. Need to declare the difference clearly

Reply:
agreed, release note should be required

Reply:
@hzxuzhonghu is the release note good enough?  or do you want further documentation/examples?

Reply:
Looks good to me